src/services/__mocks__/workspace.ts

```diff
@@ -15,20 +15,33 @@ import { WorkspaceStatus } from '../workspaceStatus';
 export const createFakeWorkspace = (
   workspaceId: string,
   workspaceName: string,
+  namespace = 'admin',
   runtime?: che.WorkspaceRuntime,
 ): che.Workspace => {
   return {
     id: workspaceId,
     attributes: {
       infrastructureNamespace: 'che',
     },
-    status: WorkspaceStatus[WorkspaceStatus.STOPPED],
+    status: runtime !== undefined ? WorkspaceStatus[WorkspaceStatus.RUNNING] : WorkspaceStatus[WorkspaceStatus.STOPPED],
```

Comment:
For such general thing I'd prefer to avoid any implicit things in creating stubs. In this case I'd throw an exception:

```typescript
if (runtime && WorkspaceStatus[WorkspaceStatus.Running] === 0) {
  throw new Error('Failed creating a stub workspace. Workspace runtime object is only combined with "RUNNING" status');
}
```

It allows us to set workspace runtime object and workspace status separately cause for the most test cases we need either a workspace runtime object or a workspace status. And there are very few cases where they are needed simultaneously in one test case.

Reply:
Ok. I have fixed it.